nautobot/ui/src/components/Layout.js

```diff
@@ -23,6 +55,35 @@ export default function Layout({ children }) {
         document.location.reload(true);
     }
 
+    function changeFeedbackInputValue(e) {
+        setFeedbackFormState((state) => ({
+            ...state,
+            [e.target.name]: e.target.value,
+        }));
+    }
+
+    function handleLeaveFeedbackSubmit(e) {
+        e.preventDefault();
+        const formData = {
+            user_agent: navigator.userAgent,
+            view_name: window.location.pathname,
+            nautobot_version: packageJson.version,
+            ...feedbackFormState,
+        };
+        const url = "https://nautobot.cloud/api/nautobot/feature-request/";
+        axios
+            .post(url, formData)
+            .then((response) => {
+                onClose();
+                feedbackToast({ title: "Feedback sent!", status: "success" });
+            })
+            .catch((err) => {
+                console.log(err);
+                onClose();
+                feedbackToast({ title: "Feedback not sent!", status: "error" });
```

Comment:
What should the user do about this? Can we provide any insight to the user as to why the error occurred? Maybe just something like `"Sorry, an error occurred when sending your feedback. Please try again later."`?